src/views/microdeposits/AccountInfo.js

```diff
@@ -96,24 +96,22 @@ export const AccountInfo = (props) => {
                 focus === AccountFields.ACCOUNT_TYPE &&
                 accountType === ReadableAccountTypes.CHECKING
               }
-              checked={accountType === ReadableAccountTypes.CHECKING}
               id={AccountTypeLabels[ReadableAccountTypes.CHECKING]}
               label={AccountTypeLabels[ReadableAccountTypes.CHECKING]}
               name="accountType"
               onChange={() => setAccountType(ReadableAccountTypes.CHECKING)}
-              style={styles.selectBox}
+              selected={accountType === ReadableAccountTypes.CHECKING}
               value={AccountTypeLabels[ReadableAccountTypes.CHECKING]}
             />
             <SelectionBox
               autoFocus={
                 focus === AccountFields.ACCOUNT_TYPE && accountType === ReadableAccountTypes.SAVINGS
               }
-              checked={accountType === ReadableAccountTypes.SAVINGS}
               id={AccountTypeLabels[ReadableAccountTypes.SAVINGS]}
               label={AccountTypeLabels[ReadableAccountTypes.SAVINGS]}
```

Comment:
The AccountInfo page does not show the text.  Looks like it's because label is being used instead of `message`

<img width="377" height="161" alt="image" src="https://github.com/user-attachments/assets/b22ab079-037b-4c5e-a6c9-b8ca0e1108f7" />

Reply:
The label should now display as expected:

<img width="543" height="552" alt="Screenshot 2025-08-07 at 15 17 28" src="https://github.com/user-attachments/assets/299b57df-bca5-45d3-b3cd-7339f5022ee7" />